package.json

```
@@ -32,10 +32,12 @@
     "package.json": "sort-package-json"
   },
   "dependencies": {
+    "@openzeppelin/contracts": "^5.0.0",
```

Remove the `^` from the dependencies:
```suggestion
    "@openzeppelin/contracts": "5.0.0",
```